andela/Ah-frontend-guardians
# #163943003 Create navbar and footer components

#### What does this PR do?
This PR provides a navigation bar and footer for all the pages that need them.

#### Description of tasks to be completed?
The navigation bar should be changing to display the avatar in case the user is logged in and display the sign in and register links if that's not the case.

#### How can this be manually tested?
* Pull the application locally and run it using the following commands, one at a time.
```
$ git clone https://github.com/andela/Ah-frontend-guardians.git
$ git pull origin
$ git checkout ft-nav-bar-footer-163943003
$ npm install
```
* Finally run the application ` npm start`
* It should navigate automatically to the browser and displays a page that is in this screenshots.
* The logged in status is set to `true` by default so please checkout the appropriate screenshot for reference.
* To change the status to `false`, in your root folder, navigate to `src` folder and then to the `reducers` folder inside which you navigate to the `navBarReducer.js` file and change `isLoggedIn` state to `false`. This should re-render and give back the output we expected in the respective screenshot below.

#### What are the relevant pivotal tracker stories?
[#163943003](https://www.pivotaltracker.com/story/show/163943003)

#### Screenshots
##### When the user's logged in status is true.
![image](https://user-images.githubusercontent.com/38134382/52961494-20407f00-33ac-11e9-8a37-4656c16e1dc5.png)

##### When the user's logged in status is false.
![image](https://user-images.githubusercontent.com/38134382/52961523-30f0f500-33ac-11e9-9c5e-2292df575a57.png)
